components/HomepageNavigation/index.js

```diff
@@ -0,0 +1,19 @@
+import { NavButton, NavContainer, HomeIcon, LoginIcon } from "./styles";
+import Link from "next/link";
+
+export default function HomePageNavigation() {
+  return (
+    <NavContainer>
+      <Link href="/">
+        <NavButton>
+          <HomeIcon />
+        </NavButton>
+      </Link>
+      <Link href="/login">
+        <NavButton>
+          <LoginIcon />
+        </NavButton>
+      </Link>
+    </NavContainer>
```

Comment:
you dont need to but a button inside a Link component, this is very confusing and bad a11y. Next Suggests a way like this: 
```
 
function Home() {
  return (
    <ul>
      <li>
        <Link href="/">Home</Link>
      </li>
      <li>
        <Link href="/about">About Us</Link>
      </li>
      <li>
        <Link href="/blog/hello-world">Blog Post</Link>
      </li>
    </ul>
  )
}
 
export default Home
```

Reply:
Hi Julia, to be honest I am not convinced. This is what I learned in my course. Can you explain further why this is not possible?

Reply:
Who told you that? 😀😀😀
You are mixing two principles. Either use a button OR use a link. The reason is accessabillity: They are semantically different and screenreaders / searchengines couldn't interpret a button inside a link. 
And I cannot see any reason for your solution. You can style the Link like a button and wrap it around the icon.

Reply:
I wish I could remember 😂  I think I heard it at some point from a coach. Maybe it was also said that it should NOT be done that way. Ooops.

Reply:
How do you like this?  
```
import { NavButton, NavContainer, HomeIcon, LoginIcon } from "./styles";
import { useRouter } from "next/router";

export default function HomePageNavigation() {
  const router = useRouter();
  const handleHomeClick = () => {
    router.push("/");
  };
  const handleLoginClick = () => {
    router.push("/login");
  };
  return (
    <NavContainer>
      <NavButton onClick={handleHomeClick}>
        <HomeIcon />
      </NavButton>
      <NavButton onClick={handleLoginClick}>
        <LoginIcon />
      </NavButton>
    </NavContainer>
  );
}
```